others/sko/css/style1.css

```diff
@@ -0,0 +1,215 @@
+body {
+    font: 14px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
+    background: #080e24 url(http://neography.com/experiment/circles/solarsystem/bg.jpg) repeat;
```

Comment:
you can probably just use the background color hex code instead of using the jpg